appmap/_implementation/event.py

```diff
@@ -297,13 +305,21 @@ def __init__(self, parent_id, elapsed, return_value):
 class HttpResponseEvent(ReturnEvent):
     __slots__ = ['http_server_response']
 
-    def __init__(self, status_code, mime_type, **kwargs):
+    def __init__(self, status_code, headers=None, **kwargs):
         super().__init__(**kwargs)
-        self.http_server_response = {
-            'status_code': status_code,
-            'mime_type': mime_type
+
+        response = {
+            'status_code': status_code
```

Comment:
FYI, the same PR I linked above fixes `status` -> `status_code` in Ruby.

Reply:
I didn't even notice it was broken...